public/app/features/dashboard/components/AddPanelButton/AddPanelMenu.tsx

```diff
@@ -23,7 +24,7 @@ export const AddPanelMenu = ({ dashboard }: Props) => {
     <Menu>
       <Menu.Item
         key="add-visualisation"
-        label="Visualization"
+        label={t('dashboard.toolbar.add-menu.visualization', 'Visualization')}
```

Comment:
Could we reduce this one (and the other labels from the menu) down to 3 segments? We are trying to keep them shorter as per the [documentation](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention) . Maybe it can be just `dashboard.toolbar.add-menu-visualization`?

Reply:
The convention we're trying to stick to is to constrain these two three segments, something like `feature.sub-are.message`. It's all very subjective and sometimes tricky to figure out what the best approach is, but would suggest something like (on top of joao's suggestion):

 - `dashboard.toolbar.add-visualization`
 - `dashboard-toolbar.add-menu.visualization`
 - `dashboard.add-menu.vizualisation` <- I think I like this the most because it because represents the conceptual hierarchy, rather than the technical implementation of where we added the add menu. It's very plausible that we might move the add menu out of the toolbar, and then it would be weird if the message id still says it's in the toolbar